### ISSUE TITLE
Improve black theme

### ISSUE THREAD

**Issue by smichel17** [mpcjanssen/simpletask-android#694]
Two main issues:

1. The contrast is a bit high.
2. Drawers need a lighter background, so they're a little more distinct from the list. Or an outline.

**Comment by ghubby123** [comment]
I disagree with these proposed changes.  The black theme is perfect as it is.  If you want non-black theming/coloring, then those changes should be a different theme altogether.

**Comment by mpcjanssen** [comment]
I agree. The point of the black theme is to use amoled black as much as possible.

**Comment by smichel17** [comment]
There are two points here:

>1. The contrast is a bit high.

The solution here is to use an off-white for text instead of `#FFF`. That will have no impact on battery life (or very slightly increased battery life).

>2. Drawers need a lighter background, so they're a little more distinct from the list. **Or an outline.**

When you open a drawer, the list dims behind it. In the light or dark themes, this makes it easy to tell where the drawer ends, even though the drawer background is the same color as the list background.

With the black theme, both backgrounds are already black, so it's difficult to tell where the drawer ends. This is especially problematic when the active filter only has a few items showing.

Emphasis added this time around. A couple px gray border on the edge of the drawer, so you can tell where the drawer ends, is all that's needed here. Actually, probably a better solution would be changing the shadows to be grey instead of black. That way we can also make the app bar AMOLED black instead of 95% gray, too.
